cnd/src/swap_protocols/halight.rs

```diff
@@ -243,7 +263,23 @@ where
     })
 }
 
-#[derive(Clone, Copy, Debug)]
+#[derive(Copy, Clone, Debug)]
 pub struct Params {
-    pub secret_hash: SecretHash,
+    pub ethereum_identity: EthereumIdentity,
+    pub ethereum_absolute_expiry: Timestamp,
+    pub lightning_identity: identity::Lightning,
+    pub lightning_cltv_expiry: Timestamp,
+    pub lightning_amount: asset::Bitcoin,
```

Comment:
:mega: We are within the `halight` scope here, the `lightning_` prefix is just noise in this module.